profiler/src/profile_batched_gemm_bias_gelu_gemm_bias.cpp

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#include <iostream>
+#include <numeric>
+#include <initializer_list>
+#include <cstdlib>
+
+#include "profiler/include/profile_batched_gemm_bias_gelu_gemm_bias_impl.hpp"
+
+using F16 = ck::half_t;
+using F32 = float;
+
+using Row = ck::tensor_layout::gemm::RowMajor;
+using Col = ck::tensor_layout::gemm::ColumnMajor;
+
+int profile_batched_gemm_bias_gelu_gemm_bias(int argc, char* argv[])
+{
+    enum struct GemmMatrixLayout
+    {
+        MK_NK_MN_ON_MO, // 0
+    };
+
+    enum struct GemmDataType
+    {
+        F16_F16_F16_F16_F16, // 0
```

Comment:
Same as above. There should only be 4 matrices concerned.

Reply:
done, +bias